src/API/Management/Clients.php

```diff
@@ -20,7 +20,7 @@ class Clients extends GenericResource
      *
      * @param null|string|array $fields         - Fields to include or exclude from the result.
      * @param null|boolean      $include_fields - True to include $fields, false to exclude $fields.
-     * @param integer           $page           - Page number to get, zero-based.
+     * @param null|integer      $page           - Page number to get, zero-based.
```

Comment:
Null to not request pagination

Reply:
@lbalmaceda - That's only for legacy clients, correct? We don't want to force pagination but don't we also want to encourage folks to use it? Seems confusing to add that with or without a note about legacy clients.

Reply:
@joshcanhelp you're right :)